Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aderyn"
-version = "0.0.7"
```

Comment:
Can you bump back down to 0.0.7? PRs are merged into `dev`, and we cut the release from there. It just so happens that I've bumped this one early, so the next release is 0.0.7.

Reply:
Got it ! I bumped it back down